enderio-base/src/main/java/com/enderio/base/common/item/tool/ElectromagnetItem.java

```diff
@@ -81,4 +81,14 @@ protected void onTickWhenActive(Player player, ItemStack pStack, Level pLevel, E
             }
         }
     }
+
```

Comment:
these seem good to have, just mention it in the commit as well?

Reply:
How can I do this now? I don't think I can modify the commit message can I?

Reply:
just in the description of the pr would be fine, it's a small thing